docs/application/environment.md

```diff
@@ -126,8 +129,28 @@ Stop the environment, the system will stop the resources in the environment in s
 
 ![app-env-stop](/img/v0.5.0/application/environment/env-stop.png)
 
-
 ## Delete Environment
 
 1. Click `Application Management` -> `Environment` in the navigation bar to enter the environment list.
 2. Ensure no resources depend on this environment, select the environment you need to delete and click `Delete` to complete the deletion.
+
+## Set Default Environment
+
+The default environment accessed after logging in for the first time in Walrus is the `local` environment under the `default` project. You can also customize the default environment:
```

Comment:
The statement seems to be wrong for non-admin users that has no access to the default project